cyclonedx-lib/build.xml

```diff
@@ -69,6 +69,31 @@
                 </copy>
         </target>
 
+        <target name="buildSignSBOM">
```

Comment:
Above "clone-and-build-openkeystore" target is a duplicate of "build-openkeystore" further below... delete this one ^